uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/apis/work-pattern-api_test.js

```diff
@@ -53,8 +53,9 @@ define([
 
     describe('getCalendar()', function () {
       var workPatternPromise;
-      var dummyContactId = 1;
-      var dummyPeriodId = 2;
+      var contactId = 1;
```

Comment:
contactId in other code places is a String as far as I remember, so should it be a string here as well? Should we check for a consistency in the code, test and mocks?

Side note: we also use toEqual, which should be avoided I think since toBe is more secure.